piker/ui/_style.py

```diff
@@ -49,11 +51,25 @@ class DpiAwareFont:
 
     def __init__(
         self,
-        # TODO: move to config
         name: str = 'Hack',
         font_size: str = 'default',
 
     ) -> None:
+        self._custom_ui = False
+        # Read preferred font size from main config file if it exists
+        conf, path = config.load('conf', touch_if_dne=True)
+        ui_section = conf.get('ui')
```

Comment:
@ebisu4 only thing i would suggest with this code is add some type annotations:

```python
ui_section: dict = conf.get('ui')
...

name: str = 'Hack'
```

etc..